src/index.js

```diff
@@ -0,0 +1,164 @@
+// import _ from 'lodash';
+/* eslint-disable prefer-destructuring */
```

Comment:
You don't need this comment here, since by removing it no issue is found for the eslint.